worker/codalabworker/docker_utils.py

```diff
@@ -80,6 +80,7 @@ def get_nvidia_devices():
         cuda_image, nvidia_command, runtime=NVIDIA_RUNTIME, detach=False, stdout=True, remove=True
     )
     # Get newline delimited gpu-index, gpu-uuid list
+    output = output.decode('utf-8')
```

Comment:
I think this should be enough (to be consistent with the style of our other `decode()` calls):
```suggestion
    output = output.decode()
```